docs/application/dotnet/tutorials/library/library-list.md

```diff
@@ -0,0 +1,57 @@
+
+# Third Party Libraries for Tizen .NET Application
+
+In this list, you can find the libraries organized by purpose.
+
+- [Hardware](#hardware)
+- [Messaging](#messaging)
+- [Multimedia](#multimedia)
+- [Security](#security)
+- [System](#system)
+- [User interface](#user-interface)
+
+## Hardware
+
+*Libraries for programming with hardware.*
+
+* [Barcode Scanner](https://github.com/Redth/ZXing.Net.Mobile) - Scan and create barcodes with ZXing.NET.Mobile.
+* [Battery Status](https://github.com/jamesmontemagno/BatteryPlugin) - Gather battery level, charging status, and type.
+* [Connectivity](https://github.com/jamesmontemagno/ConnectivityPlugin) - Get network connectivity info such as type and if connection is available.
+* [Device Info](https://github.com/jamesmontemagno/DeviceInfoPlugin) - Properties about device such as OS, Model, and Id.
+* [Device Motion](https://github.com/rdelrosario/xamarin-plugins/tree/master/DeviceMotion) - Provides access to Accelerometer, Gyroscope, Magnetometer, and Compass.
+
+## Messaging
+
+*Libraries for manipulating message.*
+
+* [Messaging](https://github.com/cjlotz/Xamarin.Plugins) - Make phone call, send sms, and send e-mail
+
+## Multimedia
+
+*Libraries for manipulating multimedia.*
+
+* [FFImageLoading](https://github.com/molinch/FFImageLoading) - Image loading with caching, placeholders, transformations and more
+* [SimpleAudioPlayer](https://github.com/adrianstevens/Xamarin-Plugins/tree/master/SimpleAudioPlayer) - SimpleAudioPlayer plays local files and audio data as a stream. This allows you to store audio data in a portable class library and play it on all supported platforms.
+* [SkiaSharp](https://github.com/mono/SkiaSharp) - The Skia 2D Graphics library from Google exposed to .NET languages and runtimes across the board
+* [Media](https://github.com/jamesmontemagno/MediaPlugin) - Take or pick photos and videos.
+* [Media Manager](https://github.com/martijn00/XamarinMediaManager) - Playback for Audio.
+
+## Security
+
+*Libraries for manipulating secure storage.*
+
+* [Secure Storage](https://github.com/sameerkapps/SecureStorage) - Provides secure storage for key value pairs Data
```

Comment:
Please add a period at the end of the sentence. Why “Data”? why not “data”?
The sentence is not clear, please explain.

Reply:
new sentence:
Provides secure storage for key-value pairs. It can be used to Securely store sensitive data strings such as password, session token, credit card number etc.

Reply:
Suggestion:
Provides secure storage for key-value pairs. It can be used to securely store sensitive data strings such as password, session token, credit card number, and so on.

Reply:
done